taipy/core/scenario/_scenario_manager.py

```diff
@@ -267,8 +267,16 @@ def _get_all_by_cycle(cls, cycle: Cycle) -> List[Scenario]:
         return cls._get_all_by(filters)
 
     @classmethod
-    def _get_primary_scenarios(cls) -> List[Scenario]:
-        return [scenario for scenario in cls._get_all() if scenario.is_primary]
+    def _get_primary_scenarios(
+        cls, sorted: Optional[bool] = False, sort_key: Optional[Literal["name", "id", "creation_date", "tags"]] = "name"
+    ) -> List[Scenario]:
+        scenarios = [scenario for scenario in cls._get_all() if scenario.is_primary]
+        if sorted:
+            if sort_key in ["name", "id", "creation_date", "tags"]:
+                scenarios.sort(key=lambda x: getattr(x, sort_key))
+            else:
+                scenarios.sort(key=lambda x: x.name)
```

Comment:
What do you think about adding `descending` parameter with default value is False, and if True then we sort with reverse=True?

Reply:
Indeed, sorting in descending order is a valuable feature, and it can be easily accomplished by using the reverse=True parameter, as you proposed. So, this addition sounds good.